pkg/plugins/platform/local_test.go

```diff
@@ -292,6 +292,27 @@ func TestLocalStop(t *testing.T) {
 	}
 }
 
+// TestDescribeStopped tests that the describe command works properly on a stopped site.
+func TestDescribeStopped(t *testing.T) {
+	assert := assert.New(t)
+	app, err := GetPluginApp("local")
+	assert.NoError(err)
+
+	for _, site := range TestSites {
+		cleanup := site.Chdir()
+
+		testcommon.ClearDockerEnv()
+		err := app.Init(site.Dir)
+		assert.NoError(err)
+
+		out, err := app.Describe()
+		assert.NoError(err)
+
+		assert.Contains(out, "stopped", "describe correctly reports stopped on a non-running site.")
```

Comment:
Completely correct that it is only output on failure. I do always struggle with the voice for these messages. 

I usually phrase it as what I expected to happen since it would be output next to a failure warning.

i.e. `Failed: describe correctly reports stopped on a non-running site` vs `Failed: describe did not correctly report stopped on a non-running site`

I know @rfay has also expressed some uncertainty for these messages. I'm probably OK with either direction, as long as we adopt one and try our best to stick to it.